docs/user/site.rst

```diff
@@ -133,12 +133,14 @@ wifi24 \: optional
     ``ap`` holds the client network configuration.
     To create an unencrypted client network, a string named ``ssid`` which sets the
     interface's ESSID is required. This is the wireless network clients connect to.
-    For an OWE secured network, the ``owe_ssid`` string has to be set. It sets the
-    SSID for the opportunistically encrypted wireless network, to which compatible
-    clients can connect to.
-    To utilize the OWE transition mode, ``owe_transition_mode`` has to be set to true.
-    Note that for the transition mode to work, both ``ssid`` as well as ``owe_ssid``
-    have to be enabled.
+    For an OWE secured network, the ``owe_ssid`` string has to be set which sets the
+    SSID for the opportunistically encrypted wireless network (to which compatible
+    clients can connect to) and the ``wireless-encryption-wpa3`` feature has to be
+    added to ``GLUON_FEATURES_standard`` in ``site.mk`` (which will enable it only
+    for non-tiny devices).
+    To disable the OWE transition mode, ``owe_transition_mode`` has to be set to
+    ``false``. Note that for the transition mode to work, both ``ssid`` as well
+    as ``owe_ssid`` have to be enabled.
```

Comment:
Why is this rewrite necessary? The default value (if absent) for the transition mode is false anyways, while this rewrite implies it's true by default.

Just add the required feature or explain why you feel this is necessary.

Reply:
Yes, I changed the default to true

Reply:
I reverted the default change